From d827bc858a98595f6a92150829aa2d6c771226a4 Mon Sep 17 00:00:00 2001
From: "flinux@vongellhorn.ch" <flinux@vongellhorn.ch>
Date: Tue, 2 Apr 2019 16:32:33 +0200
Subject: [PATCH 1/1] default-config

 - Logout timer to 10s
 - Show Date by default
 - Show date in format day month year

---
 applets/digital-clock/package/contents/config/main.xml | 8 ++++----
 lookandfeel/contents/logout/Logout.qml                 | 2 +-
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/applets/digital-clock/package/contents/config/main.xml b/applets/digital-clock/package/contents/config/main.xml
index 00560ed..64d2fc8 100644
--- a/applets/digital-clock/package/contents/config/main.xml
+++ b/applets/digital-clock/package/contents/config/main.xml
@@ -16,15 +16,15 @@
     </entry>
     <entry name="showDate" type="Bool">
       <label>Whether the date should be shown next to the clock.</label>
-      <default>false</default>
+      <default>true</default>
     </entry>
-    <entry name="dateFormat" type="string">
+    <entry name="custom" type="string">
       <label>The date format to display. Options are: shortDate, longDate, isoDate or custom.</label>
-      <default>shortDate</default>
+      <default>customDateFormat</default>
     </entry>
     <entry name="customDateFormat" type="string">
       <label>Custom date format string.</label>
-      <default>ddd d</default>
+      <default>d MMM yyyy</default>
     </entry>
     <entry name="fontFamily" type="string">
       <label>Font family. e.g "arial". The system font is used if this is not set.</label>
diff --git a/lookandfeel/contents/logout/Logout.qml b/lookandfeel/contents/logout/Logout.qml
index f26b156..7b59f12 100644
--- a/lookandfeel/contents/logout/Logout.qml
+++ b/lookandfeel/contents/logout/Logout.qml
@@ -54,7 +54,7 @@ PlasmaCore.ColorScope {
         root.suspendRequested(4);
     }
  
-    property real timeout: 30
+    property real timeout: 10
     property real remainingTime: root.timeout
     property var currentAction: {
         switch (sdtype) {
-- 
2.21.0

